engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -336,7 +336,7 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
         check {
           status should be(StatusCodes.OK)
           val decoder: Decoder = Gzip
-          val result = Await.result(Unmarshal(decoder.decode(response)).to[JsObject], 1.second)
+          val result = Await.result(Unmarshal(decoder.decodeMessage(response)).to[JsObject], 1.second)
```

Comment:
IIRC there was a reason why I couldn't do that in the first place (I *think* I used `responseAs` elsewhere in the file, I definitely played around with it). And by "couldn't", i probably mean "that was going to be a giant pain in the ass for some reason I didn't want to deal with it at the time". 

I'll take another poke.